.changeset/chubby-cameras-argue.md

```diff
@@ -0,0 +1,11 @@
+---
+"@biomejs/biome": major
+---
+
+Removed `--config-path` argument from `biome lsp-proxy` and `biome start` commands.
```

Comment:
If possible, it would be nice to explain why the option was removed. What were the limitations/constraints